web/src/components/dashboard/OpportunityCard.tsx

```diff
@@ -72,6 +74,12 @@ export const OpportunityCard = (props: Props): ReactElement => {
       .update();
   };
 
+  const routeToPage = (): void => {
+    const url = `/${props.urlPath}/${props.opportunity.urlSlug}`;
```

Comment:
[sand] it'd be nice to have a test that the router push is called with the correct url. we weren't doing it before (probably because it's a little more annoying to test when it's an `a` href) but it seems like a good opportunity to add test coverage